packages/ui/components/UserProfile/UserProfile.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { User } from '@codement/api';
+import { Size } from '../../types/styled';
+
+export interface UserProfileProps extends React.HTMLAttributes<HTMLDivElement> {
+  user: User | string;
+  size?: Size | number;
+}
+
+const Image = styled.img<{size?: Size | number}>`
+  height: ${p => p.theme.size(p.size)};
+  width: ${p => p.theme.size(p.size)};
+  border-radius: 50%;
```

Comment:
Yeah TBH, I feel like because it's the same value every time it doens't really matter haha. It's more "technically" correct, but in reality it's kinda the same.

If it was a CUSTOM value, it would be a completely different story, but because a circle is ALWAYS 50%...🤷‍♂️